spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -37,46 +58,81 @@
       stub_analytics
     end
 
-    it 'returns OK and logs an event with a correct secret key and body' do
-      request.headers['Authorization'] = socure_secret_key
-      post :create, params: webhook_body
-
-      expect(response).to have_http_status(:ok)
-      expect(@analytics).to have_logged_event(
-        :idv_doc_auth_socure_webhook_received,
-        created_at: '2020-01-01T00:00:00Z',
-        customer_user_id: '123',
-        event_type: 'TEST_WEBHOOK',
-        reference_id: 'abc',
-        user_id: '123',
-      )
-    end
+    context 'webhook authentication' do
+      it 'returns OK and logs an event with a correct secret key and body' do
+        request.headers['Authorization'] = socure_secret_key
+        post :create, params: webhook_body
 
-    it 'returns OK with an older secret key' do
-      request.headers['Authorization'] = socure_secret_key_queue.last
-      post :create, params: webhook_body
+        expect(response).to have_http_status(:ok)
+        expect(@analytics).to have_logged_event(
+          :idv_doc_auth_socure_webhook_received,
+          created_at: '2020-01-01T00:00:00Z',
+          customer_user_id: '123',
+          event_type: 'TEST_WEBHOOK',
+          reference_id: 'abc',
+          user_id: '123',
+        )
+      end
 
-      expect(response).to have_http_status(:ok)
-    end
+      it 'returns OK with an older secret key' do
+        request.headers['Authorization'] = socure_secret_key_queue.last
+        post :create, params: webhook_body
 
-    it 'returns unauthorized with a bad secret key' do
-      request.headers['Authorization'] = 'ABC123'
-      post :create, params: webhook_body
+        expect(response).to have_http_status(:ok)
+      end
 
-      expect(response).to have_http_status(:unauthorized)
-    end
+      it 'returns unauthorized with a bad secret key' do
+        request.headers['Authorization'] = 'ABC123'
+        post :create, params: webhook_body
 
-    it 'returns unauthorized with no secret key' do
-      post :create, params: webhook_body
+        expect(response).to have_http_status(:unauthorized)
+      end
 
-      expect(response).to have_http_status(:unauthorized)
-    end
+      it 'returns unauthorized with no secret key' do
+        post :create, params: webhook_body
 
-    it 'returns bad request with no event in the body' do
-      request.headers['Authorization'] = socure_secret_key
-      post :create, params: {}
+        expect(response).to have_http_status(:unauthorized)
+      end
+
+      it 'returns bad request with no event in the body' do
+        request.headers['Authorization'] = socure_secret_key
+        post :create, params: {}
 
-      expect(response).to have_http_status(:bad_request)
+        expect(response).to have_http_status(:bad_request)
+      end
+    end
+
+    context 'DOCUMENTS_UPLOADED webhook' do
```

Comment:
LGTM 👍🏿 

nit ... this seems like the same context as below `context 'when DOCUMENTS_UPLOADED event received' do` which is duplicative ... 
could you move into your tests to the existing context?